platform/linux-generic/odp_timer.c

```diff
@@ -1413,6 +1413,11 @@ int odp_timer_init_global(const odp_init_t *params)
 	const char *conf_str;
 	int val = 0;
 
+	if (params && params->not_used.feat.timer) {
+		ODP_DBG("Timers disabled\n");
+		return 0;
```

Comment:
Since `odp_timer_term_global()` assumes that `timer_global == NULL` if timers are not being used, you should explicitly set `timer_global` to `NULL` here to avoid problems if an ODP instance with timers is followed by an ODP instance without timers.

Reply:
Thanks, will fix. Thinking about it we may have the same assumption (static pointers initialized always to NULL) also in other places.